app-text/sigil/files/sigil-0.9.13-proper-gumbo-install.patch

```diff
@@ -0,0 +1,24 @@
+--- a/internal/gumbo/CMakeLists.txt
++++ b/internal/gumbo/CMakeLists.txt
+@@ -40,4 +40,5 @@ endif()
+ 
+ if( UNIX AND NOT APPLE )
+ 	set( CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -std=c99" )
++    install( TARGETS sigilgumbo DESTINATION ${CMAKE_INSTALL_PREFIX}/${LIBDIR}/sigil/ )
```

Comment:
Why are you moving it? Also, is it building a bundled gumbo instead of using dev-libs/gumbo? Also, mismatched indentation.

Reply:
The gumbo is patched with patches no accepted by upstream (found in wiki of Sigil). About the install patch, changed.